ui/src/app/pages/data/concept/concept-set-details.tsx

```diff
@@ -264,6 +264,15 @@ export const ConceptSetDetails = fp.flow(withUrlParams(), withCurrentWorkspace()
       navigateByUrl('workspaces/' + workspace.namespace + '/' + workspace.id + '/data/concepts' + queryParams);
     }
 
+    get displayDomainName() {
+      const {conceptSet} = this.state;
+      return conceptSet.domain === Domain.PHYSICALMEASUREMENT
+          ? 'Physical Measurements' :
+          conceptSet.domain === Domain.SURVEY
+              ? fp.capitalize(conceptSet.domain.toString() + '-' + conceptSet.survey) :
+              fp.capitalize(conceptSet.domain.toString()) ;
```

Comment:
<img width="490" alt="Screen Shot 2020-10-26 at 3 17 15 PM" src="https://user-images.githubusercontent.com/40036095/97223866-5992da00-179e-11eb-96b3-758bc7e80ea7.png">

Can you add a space on either side of '-' and capitalize the survey name? 
Seems that `fp.capitalize` only affects the first word of a string, might be better to use `textTransform: 'capitalize'` on the div.
```suggestion
              ? conceptSet.domain.toString() + ' - ' + conceptSet.survey
              : conceptSet.domain.toString() ;
```

Reply:
Done and updated screenshot